internal/transport/http2_server.go

```diff
@@ -1042,6 +1042,18 @@ func (t *http2Server) writeHeaderLocked(s *Stream) error {
 	return nil
 }
 
+type NoopFunc func()
+
+var noop NoopFunc = func() {
+	// Default noop function does nothing
+}
+
+func setRstAndCallNoop(rst *bool) {
```

Comment:
this is not needed either. Please remove

Reply:
done